README.md

```diff
@@ -1,19 +1,31 @@
 # data-and-algorithms
 Let's go over data structures and algorithms together! 
-We will use Cracking the Coding Interview as reference.
+We will use [Cracking the Coding Interview Book](http://www.crackingthecodinginterview.com/) as reference.
 
 Feel free to add your solutions in your channel of your choice 😃
 
+Run this script to create the folder structure for your favorite programming language
+`sh createFolders.sh  <Programming Language Name>`
```

Comment:
tiny naming nit: in POSIX, these are called [directories](https://en.wikibooks.org/wiki/A_Quick_Introduction_to_Unix/Directory_Structure), not folders.